src/compiler/checker.ts

```diff
@@ -34845,18 +34845,20 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
         return restParameter.escapedName;
     }
 
-    function getParameterIdentifierNameAtPosition(signature: Signature, pos: number): [parameterName: __String, isRestParameter: boolean] | undefined {
+    function getParameterIdentifierInfoAtPosition(signature: Signature, pos: number): [parameter: Identifier, parameterName: __String, isRestParameter: boolean] | undefined {

```

Comment:
I feel like it is ridiculous that this was ever a tuple. This should just be an object at this point.